### PR TITLE
Beginning of french translation

### PR DESCRIPTION
**Pull request by jydidier**
Hi!
I started a french translation of your awesome project two weeks ago. 
So far, 3 articles are translated. I hope many will come soon.

Best regards, 
Jean-Yves

### CODE REVIEW

**Comment by greggman** [comment]
Wow! Awesome!

File: threejs/lessons/fr/threejs-fundamentals.md [hunk 1]
```diff
@@ -0,0 +1,493 @@
...
+
+Ceci est le premier article d'une série consacrée à three.js.
+[Three.js](https://threejs.org) est une bibliothèque 3D qui a pour objectif
+de rendre aussi facile que possible l'inclusion de contenu 3D dans une page web.
+
+Three.js est souvent confondu avec WebGL puisque la plupart du temps, mais
+pas toujours, elle exploite WebGL pour dessiner en 3D.
+[WebGL est un système très bas niveau qui ne dessine que des points, des lignes et des triangles](https://webglfundamentals.org).
+Faire quelque chose d'exploitable avec WebGL requiert une certaine quantité de code
+et c'est là que three.hs intervient. Elle prend en charge des choses
```
Comment (Lecrapouille) [inline threejs/lessons/fr/threejs-fundamentals.md:13]: three.js

File: threejs/lessons/fr/threejs-fundamentals.md [hunk 2]
```diff
@@ -0,0 +1,493 @@
...
+et de les connecter. Voici un diagramme qui représente une application
+three.js de petite taille:
+
+<div class="threejs_center"><img src="resources/images/threejs-structure.svg" style="width: 768px;"></div>
+
+Voici ce qui est à remarquer dans le diagramme ci-dessus :
+
+* Il y a un `Renderer`. C'est sans doute l'objet principal de three.js. Vous passez
+  une `Scene` et une `Camera` à un `Renderer` et il effectue le rendu (dessine) de la
+  partie de la scene 3D qui est à l'intérieur de l'espace visible (en réalité une pyramide tronquée ou *frustum*)
```
Comment (Lecrapouille) [inline threejs/lessons/fr/threejs-fundamentals.md:44]: scène 3D

File: threejs/lessons/fr/threejs-fundamentals.md [hunk 3]
```diff
@@ -0,0 +1,493 @@
...
+Ensuite, nous créons une `Scene`. Dans three.js, une `Scene` est la racine 
+du graphe de scène. Tout ce que nous voulons que three.js dessine doit être ajouté
+à la scene. Cela est davantage détaillé dans [un futur article sur le fonctionnement des scènes](threejs-scenegraph.html).
+
+```js
+const scene = new THREE.Scene();
+```
+
+Ensuite nous créons une géométrie de type `BoxGeometry` qui contient les données pour un parallélépipède.
+Quasimment tout ce que nous souhaitons afficher avec Three.js nécessite une
```
Comment (Lecrapouille) [inline threejs/lessons/fr/threejs-fundamentals.md:211]: Quasiment

File: threejs/lessons/fr/threejs-fundamentals.md [hunk 4]
```diff
@@ -0,0 +1,493 @@
...
+  const cube = new THREE.Mesh(geometry, material);
+  scene.add(cube);
+
+  cube.position.x = x;
+
+  return cube;
+}
+```
+
+Ensuite, nous l'appelerons à 3 reprises avec 3 différentes couleurs
```
Comment (Lecrapouille) [inline threejs/lessons/fr/threejs-fundamentals.md:361]: appellerons

File: threejs/lessons/fr/threejs-fundamentals.md [hunk 5]
```diff
@@ -0,0 +1,493 @@
...
+<p>
+La raison nécessitant cette structuration des dossiers est parce que les scripts
+des exemples tels que <a href="https://github.com/mrdoob/three.js/blob/master/examples/jsm/controls/OrbitControls.js"><code>OrbitControls.js</code></a>
+ont des chemins relatifs tels que
+</p>
+<pre class="prettyprint">
+import * as THREE from '../../../build/three.module.js';
+</pre>
+<p>
+Utiliser la même structure permet de s'assurer lors de l'import à la fois de
```
Comment (Lecrapouille) [inline threejs/lessons/fr/threejs-fundamentals.md:469]: importation + manque des virgules ? => lors de l'importation, à la fois de three.js et d'une autre librarie,

File: threejs/lessons/fr/threejs-fundamentals.md [hunk 6]
```diff
@@ -0,0 +1,493 @@
...
+</pre>
+<p>Cela est valable aussi lors de l'utilisation d'un CDN. Assurez vous que vos chemins versThis includes when using a CDN. Be  <code>three.modules.js</code> terminent par
+<code>/build/three.modules.js</code>. Par exemple :</p>
+<pre class="prettyprint">
+import * as THREE from 'https://unpkg.com/three@0.108.0/<b>build/three.module.js</b>';
+import {OrbitControls} from 'https://unpkg.com/three@0.108.0/examples/jsm/controls/OrbitControls.js';
+</pre>
+<p>Si vous préférez la vieille manière<code>&lt;script src="path/to/three.js"&gt;&lt;/script&gt;</code>,
+vous pouvez consulter <a href="https://r105.threejsfundamentals.org">une version plus ancienne de ce site</a>.
+Three.js a pour politique de ne pas s'embarrasser avec la rétrocompatibilité.
```
Comment (Lecrapouille) [inline threejs/lessons/fr/threejs-fundamentals.md:485]: rétro compatibilité

**Comment by Lecrapouille** [comment]
I did not notice the PR was already merged. I'll create a PR for fixing typos.

**Pull request merged**